booking-up-for-beauty/ena/booking_up_for_beauty.go

```diff
@@ -0,0 +1,47 @@
+package booking
+
+import (
+	"strconv"
+	"time"
+)
+
+// Schedule returns a time.Time from a string containing a date
+func Schedule(date string) time.Time {
+	// input format "7/25/2019 13:45:00"
+	const layout = "1/2/2006 15:04:05"
+	t, _ := time.Parse(layout, date)
```

Comment:
I'd add an utility function to parse dates and handle errors.

Reply:
You are right. Exit status should always be checked. I considered that, but since it is not explicitly defined how the errors should be handled (print an error or throw exception (if that what panic is doing)), i decided to just throw it away :-).